iq/iq.go

```diff
@@ -168,6 +169,10 @@ func New(logger *logrus.Logger, options Options) (server *Server, err error) {
 		options.TTL = time.Now().Local().Add(time.Hour * 12)
 	}
 
+	if strings.HasSuffix(options.Server, "/") {
+		options.Server = strings.TrimSuffix(options.Server, "/")
```

Comment:
Up to you but we could probably make this more robust to other weird slash issues with something like so. 

https://play.golang.org/p/vhBiCFbs-ZI

Reply:
Ehhhhhhh, in AuditJS we did something where we actually validated it as a real URI, kinda think we'd do something like this for that: https://stackoverflow.com/questions/31480710/validate-url-with-standard-package-in-go

Reply:
Oof. Nothing is ever "easy", especially the "easy" things. 
Feels like this is a step in the right direction to me.